modules/ROOT/pages/sharepoint/sharepoint-connector.adoc

```diff
@@ -157,17 +245,17 @@ To build and run this demo project, you need:
 * Anypoint Studio with at least the Mule 4.0 Runtime.
 * Microsoft SharePoint Connector v2.0.0 or higher.
 
-=== To Test the Flow
+=== Test the Flow
 
-. Import the demo project into your workspace using Anypoint Exchange or using the Import command in the File menu.
+. Import the demo project into your workspace using Anypoint Exchange or using the *File* > *Import command.
```

Comment:
```suggestion
. Import the demo project to your workspace using Anypoint Exchange or using the *File* > *Import command.
```
NOTE: there is something that needs fixing in "the *File* > *Import command". 
Maybe just a missing *
But...clicking UI elements is not using a command....